src/libsystemd-network/sd-radv.c

```diff
@@ -126,6 +126,8 @@ _public_ sd_radv *sd_radv_unref(sd_radv *ra) {
                 sd_radv_prefix_unref(p);
         }
 
+        mfree(ra->rdnss);
```

Comment:
mfree()'s only raison d'être is that it returns NULL, so that you can write `p = mfree(p)` and thus in a single line set p to NULL and free it. If you ignor its return value, it's not any better than free(), and that's what you should use then...

But then again, this doesn't really matter I figure...